File: src/main/java/me/conorthedev/mediamod/keybinds/KeybindInputHandler.java
package me.conorthedev.mediamod.keybinds;

import me.conorthedev.mediamod.config.Settings;
import net.minecraft.client.Minecraft;
import net.minecraft.util.ChatComponentText;
import net.minecraft.util.EnumChatFormatting;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
import net.minecraftforge.fml.common.gameevent.InputEvent;

/**
 * The class that handles keybind events
 */
public class KeybindInputHandler {
    /**
     * Fired when a key is pressed
     *
     * @param event - KeyInputEvent
     * @see InputEvent.KeyInputEvent
     */
    @SubscribeEvent
    public void onKeyInput(InputEvent.KeyInputEvent event) {
        if (KeybindManager.INSTANCE.disableKeybind.isPressed()) {
            if (!Settings.SHOW_PLAYER) {
                Minecraft.getMinecraft().thePlayer.addChatComponentMessage(new ChatComponentText(EnumChatFormatting.RED + "[" + EnumChatFormatting.WHITE + "MediaMod" + EnumChatFormatting.RED + "] " + EnumChatFormatting.RESET + "Player Visible"));

                Settings.SHOW_PLAYER = true;
            } else {
                Minecraft.getMinecraft().thePlayer.addChatComponentMessage(new ChatComponentText(EnumChatFormatting.RED + "[" + EnumChatFormatting.WHITE + "MediaMod" + EnumChatFormatting.RED + "] " + EnumChatFormatting.RESET + "Player Hidden"));

                Settings.SHOW_PLAYER = false;
            }
            Settings.saveConfig();
        }
    }
}

File: src/main/kotlin/dev/cbyrne/mediamod/keybinds/KeybindInputHandler.kt
package dev.cbyrne.mediamod.keybinds

import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
import net.minecraftforge.fml.common.gameevent.InputEvent

class KeybindInputHandler {
    @SubscribeEvent
    fun onKey(e: InputEvent.KeyInputEvent) {
        if(KeybindManager.disableKeybind.isPressed) {
            // todo: disable / show player
        } else if (KeybindManager.menuKeybind.isPressed) {
            // todo: show settings gui
        }
    }
}